includes/apple-exporter/components/class-audio.php

```diff
@@ -54,6 +54,11 @@ protected function build( $text ) {
 
 		$url = $match[1];
 
+		// If the URL is root-relative, make it absolute.
+		if ( 0 === strpos( $url, '/' ) ) {
```

Comment:
Not a blocker. I see you used this twice, maybe this can be a standalone function?